src/API.CRUD/Program.cs

```diff
@@ -11,6 +11,8 @@ public class Program
     {
         public static int Main(string[] args)
         {
+            // SERILOG + Staging/Production builds?

```

Comment:
For discussion

Reply:
Leaving until I update the nugets, etc to sdk 2.0.